src/WebJobs.Script/ScriptConstants.cs

```diff
@@ -124,7 +124,8 @@ public static class ScriptConstants
         public const string FeatureFlagEnableDebugTracing = "EnableDebugTracing";
         public const string FeatureFlagEnableMultiLanguageWorker = "EnableMultiLanguageWorker";
         public const string FeatureFlagEnableProxies = "EnableProxies";
-        public const string FeatureFlagEnableLinuxEPExecutionCount = "EnableLinuxFEC";
+        public const string HostingConfigEnableLinuxAppServiceExecutionCount = "EnableLinuxFEC";
```

Comment:
based on my comments above, we may want to rename this as well. E.g. EnableLinuxDetailedExecutionEvents

Reply:
I did update this to HostingConfigEnableLinuxAppServiceDetailedExecutionEvents.  
This code does not run for Linux Consumption so i wanted to try to make that clear.